library/core/src/main/java/com/google/android/exoplayer2/trackselection/AdaptiveTrackSelection.java

```diff
@@ -88,7 +88,7 @@ public AdaptiveTrackSelection createTrackSelection(TrackGroup group, int... trac
   }
 
   public static final int DEFAULT_MAX_INITIAL_BITRATE = 800000;
-  public static final int DEFAULT_MIN_DURATION_FOR_QUALITY_INCREASE_MS = 10000;
+  public static final int DEFAULT_MIN_DURATION_FOR_QUALITY_INCREASE_MS = 4000;
```

Comment:
I don't think it's a good idea just to change the default for all playbacks. This also affects on-demand playbacks, and live playbacks when the playback position is no-where near the live edge. It seems preferable to give the selector more information instead. For example if the selector were to know how close it is to the live edge, or something like that, then it could adapt its behavior accordingly.